torch/_refs/__init__.py

```diff
@@ -3146,6 +3146,33 @@ def equal(a: TensorLikeType, b: TensorLikeType) -> bool:
     return item(all(eq(a, b)))  # type: ignore[return-value]
 
 
+@register_decomposition(torch.ops.aten.norm)
```

Comment:
cc @chillee, this seems to be using different overloads than original decomposition registration, is it ok?
Also, previous decomposition dispatched to ops that could be fused by nvfuser, whereas for this one it's unclear. Is this a concern and are there any tests we can run for this?

Reply:
Yeah that's fine - this will just register it to all overloads (which is good now, since it also handles dtype).

But actually, I think this is the incorrect registration @Lezcano. I think `aten.norm` corresponds to `torch.linalg.vector_norm` :)

Reply:
You are completely right! 
Now, I assume that, since from the next PR in this stack we are going to have `norm` dispatching to `vector_norm` and `matrix_norm` and these are decomposed already, it should be fine not to register this as a decomposition, right?

Reply:
Yeah this decomposition isn't needed for `__torch_dispatch__`.